roles/network/templates/dummy.netdev.j2

```diff
@@ -1,3 +1,4 @@
 [NetDev]
 Name={{ item.1 }}
 Kind=dummy
+MTUBytes={{ network_dummy_interfaces_mtu }}
```

Comment:
Use ``network_dummy_interface_mtu`` and define this parameter in the defaults/main.yml as well.

Reply:
done